### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by HarshNarayanJha**
Added new issue templates, one for `Bug report` and another for a `Feature request`.

### CODE REVIEW

**Comment by HarshNarayanJha** [comment]
Hey @loqusion, here are the templates! Please review and see if everything seems right.

**Comment by HarshNarayanJha** [comment]
Hey @loqusion, The Templates!

**Review by loqusion — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+paste the ouput of `hyprctl version`
+```
+- **Hyprshade version**:
+```
+paste ouput of `hyprshade --version`
+```
+- **Linux Distro**: [e.g. Ubuntu, Fedora, Arch Linux]
+- **GPU (if any)**: [e.g. NVIDIA, AMD, Intel]
+
+- **Hyprshade config**: Please provide the contents of your `hyprshade.toml` file (generally at `$XDG_CONFIG_HOME/hypr/hyprshade.toml` or `~/.config/hypr/hyprshade.toml`, if not present then skip this section)
```
[the remark this answers is not in the document]
Comment (HarshNarayanJha) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- **Hyprland version**:
+```
+paste the ouput of `hyprctl version`
+```
+- **Hyprshade version**:
+```
+paste ouput of `hyprshade --version`
+```
+- **Linux Distro**: [e.g. Ubuntu, Fedora, Arch Linux]
+- **GPU (if any)**: [e.g. NVIDIA, AMD, Intel]
```
[the remark this answers is not in the document]
Comment (HarshNarayanJha) [reply .github/ISSUE_TEMPLATE/bug_report.md:36]: don't know a lot about nixpkgs and nixos though. if needed I can edit the message

- **Hyprshade Installation Method**: [e.g. pip, AUR, nixpkgs]

**Comment by HarshNarayanJha** [comment]
> Oops, didn't know you had to click a button to finish a review

Oh! No worries.

**Pull request merged**